pyproject.toml

```diff
@@ -4,7 +4,7 @@ version = "0.4.5"
 description = "Federated blocklist management for Mastodon"
 readme = "README.md"
 license = {file = "LICENSE"}
-requires-python = ">=3.6"
+requires-python = ">=3.8"
```

Comment:
Dropping Python 3.6 support (and therefore Ubuntu 18.04LTS) is really my only concern. Do we have to? 

I'm okay if we do, but we will need to make it clear to admins that this is happening so they don't break their setups with an upgrade if later versions of Python aren't available to them.

Reply:
The code today does not support Python 3.6 or 3.7: 

```
ImportError while importing test module '/Users/offby1/projects/wandering.shop/fediblockhole/tests/test_parser_rapidblockjson.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/test_parser_rapidblockjson.py:3: in <module>
    from fediblockhole.blocklists import parse_blocklist
src/fediblockhole/__init__.py:14: in <module>
    from .blocklists import Blocklist, BlockAuditList, parse_blocklist
src/fediblockhole/blocklists.py:9: in <module>
    from .const import DomainBlock, BlockSeverity, BlockAudit
src/fediblockhole/const.py:5: in <module>
    from typing import NamedTuple, Optional, TypedDict
E   ImportError: cannot import name 'TypedDict' from 'typing' (/Users/offby1/.local/share/mise/installs/python/3.7.17/lib/python3.7/typing.py)
```

It uses features that were only added in Python 3.8